api/com/dynatrace/oneagent/sdk/api/infos/TraceContextInfo.java

```diff
@@ -1,7 +1,7 @@
 package com.dynatrace.oneagent.sdk.api.infos;
 
 /**
- * Provides information about the current PurePath node using the TraceContext
+ * Provides information about a current PurePath node using the TraceContext
```

Comment:
```suggestion
 * Provides information about a PurePath node using the TraceContext
```

Reply:
Thanks for the catch. I figured I could do that small change manually :man_facepalming:  ... next time it's the script again.